### PR TITLE
Add formula for 1.3.0, 0.29.1 & 0.28.2 CLI releases

### PR DESCRIPTION
**Pull request by neel-astro**
Changes:
- Added formula for 1.3.0 CLI release & updated the astro.rb to 1.3.0 (https://github.com/astronomer/astro-cli/releases/tag/v1.3.0)
- Added formula for 0.29.1 CLI release (https://github.com/astronomer/astro-cli/releases/tag/v0.29.1)
- Added formula for 0.28.2 CLI release (https://github.com/astronomer/astro-cli/releases/tag/v0.28.2)

### CODE REVIEW

File: Formula/astro@1.3.0.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  
+    def install
+      bin.install "astro"
+    end
+  
+  
+    test do
+      system "#{bin}/astro version"
+    end
+  end
```
Comment (andriisoldatenko) [inline Formula/astro@1.3.0.rb:40]: is it automatic on manual change?
```suggestion
  end

```

File: Formula/astro@0.29.1.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    def install
+      bin.install "astro"
+    end
+  
+  
+    test do
+      system "#{bin}/astro version"
+    end
+  end
+
```
Comment (andriisoldatenko) [inline Formula/astro@0.29.1.rb:41]:
```suggestion

```

File: Formula/astro@1.3.0.rb (hunk 1 above)
Comment (neel-astro) [reply Formula/astro@1.3.0.rb]: must have been accidentally updated when verifying the file contents with the checksum file of the release

File: Formula/astro@0.29.1.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  
+    def install
+      bin.install "astro"
+    end
+  
+  
+    test do
+      system "#{bin}/astro version"
+    end
+  end
```
Comment (andriisoldatenko) [inline Formula/astro@0.29.1.rb:40]:
```suggestion
  end

```

**Pull request merged**